sklearn/feature_extraction/text.py

```diff
@@ -1005,6 +1023,9 @@ class TfidfTransformer(BaseEstimator, TransformerMixin):
     Tf is "n" (natural) by default, "l" (logarithmic) when
     ``sublinear_tf=True``.
     Idf is "t" when use_idf is given, "n" (none) otherwise.
+    The "norm" parameter provides the "how" of how each input vector is
```

Comment:
It is - I was trying to address it as it seemed like it was one of the points of discussion in the previous issue. Since norm discussed as a parameter, though, it makes more sense to discuss the default there.